Nos/Views/AppView.swift

```diff
@@ -9,23 +9,32 @@ import SwiftUI
 
 struct AppView: View {
     
+    @StateObject private var appController = AppController()
+    
     @State var path = NavigationPath()
     
     var body: some View {
-        NavigationStack(path: $path) {
-            List {
-                NavigationLink(Localized.homeFeedLinkTitle.string) {
-                    HomeFeedView()
-                }
-                NavigationLink(Localized.relaysLinkTitle.string) {
-                    RelayView()
-                }
-                NavigationLink(Localized.settingsLinkTitle.string) {
-                    SettingsView()
+        ZStack {
```

Comment:
This works but I'm used to seeing `Group {` as the generic heterogenous wrapper view.
```suggestion
        Group {
```